Documentation/docs/contributing/python_packaging.md

```diff
@@ -0,0 +1,148 @@
+Build ITK Python packages
+======================================
+
+We can generate Python wrappings and package them as Python wheels in order to conveniently distribute and use the ITK C++ library in Python scripts. In most cases community members can use the [general ITK Python distributions](https://pypi.org/project/itk/) on the Python Package Index (PyPI) for their projects.
+
+```py
+> pip install itk
```

Comment:
If we can remove `>`, when rendered there will be will a copy butten, and we can copy directly from the rendered docs.